src/common/lib/polkadot-provider/events/handlers/geneticAnalyst.js

```diff
@@ -0,0 +1,45 @@
+const defaultHandler = {
+  get: function (target, name) {
+    return Object.prototype.hasOwnProperty.call(target, name) ? target[name] : (dataEvent, value, valueMessage) => {
+      const data = dataEvent[0]
+      const id = data[value]
+      const params = { number: id }
+      const wording = "for (" + data[valueMessage].substr(0, 4) + "..." + data[valueMessage].substr(data[valueMessage].length - 4) + ")"
+      return { data, id, params, wording }
+    }
+  }
+}
```

Comment:
The logic is getting increasingly complicated, we need to add unit tests for this? Especially for the `polkadot-provider/events`.

Please make new unit test in `tests/unit/common/lib/polkadot-provider/events`.